albumentations/augmentations/transforms.py

```diff
@@ -3320,3 +3322,91 @@ def get_transform_init_args_names(self):
             "mask_fill_value",
             "random_offset",
         )
+
+
+class Defocus(ImageOnlyTransform):
+    """
+    Apply defocus transform. See arXiv:1903.12261 [cs.LG].
+
+    Args:
+        radius ((int, int) or int): range for radius of defocusing.
+            If limit is a single int, the range will be [1, limit]. Default: (3, 10).
+        alias_blur ((float, float) or float): range for alias_blur of defocusing (sigma of gaussian blur).
+            If limit is a single float, the range will be (0, limit). Default: (0.1, 0.5).
+        p (float): probability of applying the transform. Default: 0.5.
+
+    Targets:
+        image
+
+    Image types:
+        Any
+    """
+
+    def __init__(self, radius=(3, 10), alias_blur=(0.1, 0.5), always_apply=False, p=0.5):
+        super(Defocus, self).__init__(always_apply, p)
+        self.radius = to_tuple(radius, low=1)
+        self.alias_blur = to_tuple(alias_blur, low=0)
+
+        if self.radius[0] < 0:
+            raise ValueError("Parameter radius must be positive")
+
+        if self.alias_blur[0] <= 0:
+            raise ValueError("Parameter alias_blur must be non-negative")
```

Comment:
I think this checks are redundant because you described the minimum values ​​in `to_tuple`

Reply:
It's for the case when radius is set to negative value x, so tuple would be (x, 0).
On the other hand, user could set interval like (-5, 100) and this check will not allow it.